GUIDE.md

```diff
@@ -187,7 +187,7 @@ The identity also contains signatures proving possession of the id and OrbitDB p
 
 The OrbitDB public key can be retrieved with:
 ```javascript
-console.log(db.identity.publicKey)
+console.log(db.identity._publicKey)
```

Comment:
looks like there is using the getter function
https://github.com/orbitdb/orbit-db-identity-provider/blob/e93809dda6cfa5b0fd843815d2a5f8f5bc1e81d9/src/identity.js#L45-L47

Reply:
Yeah, which API do you want to expose, because the documentation specifically says that its `db.identity` which would return the getter function version.

Reply:
underscored properties arent meant to be accessed by users. i dont think those parts need to be updated.